pramen-py/src/pramen_py/metastore/reader.py

```diff
@@ -218,15 +221,17 @@ def from_config(self, config: ConfigTree):  # type: ignore
         for table in tables:
             metastore = MetastoreTable(
                 name=table.get("name", ""),
-                format=table.get("format", ""),
+                format=get_table_format_by_value(table.get("format", "")),
                 path=table.get("path", ""),
                 table=table.get("table", ""),
                 description=table.get("description", ""),
                 records_per_partition=table.get(
                     "records_per_partition", 500000
                 ),
                 info_date_settings=InfoDateSettings(
-                    column=table.get("information.date.column", ""),
+                    column=table.get(
+                        "information.date.column", "pramen_info_date"
```

Comment:
Also need to check the global info date column default. The configuration key is `pramen.information.date.column`.

e.g. 
```python
table.get("information.date.column", config.get("pramen.information.date.column", "pramen_info_date"))
```